### PR TITLE
Add support for mobile web uber as preference over RRW

### LINKED ISSUE #111: Add support for Uber Mobile Web as Fallback option
Add support for more featured mobile web option at https://m.uber.com as a preference over the Ride Request Widget.

### PR DESCRIPTION
**Pull request by tyvsmith**
Description:
Adds deeplink generation for m.uber.com and marks the APIs for the widget activity, view, and helper methods with a deprecated annotation in favor of using the more featured m.uber.com site.

Related issue(s):
Fixes #111

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # Uber Rides Android SDK (beta) [![Build Status](https://travis-ci.org/uber/rides-android-sdk.svg?branch=master)](https://travis-ci.org/uber/rides-android-sdk)
 
-Official Android SDK (beta) to support:
+Official Android SDK to support:
  - Ride Request Button
- - Ride Request Widget
+ - RIde Request Deeplinks
```
Comment (edjiang) [inline README.md]: Typo

File: rides-android/src/main/java/com/uber/sdk/android/rides/RequestDeeplink.java [hunk 2]
```diff
@@ -23,188 +23,23 @@
...
-
-    private enum LocationType {
-        PICKUP,
-        DROPOFF;
-
-        private String getUriQueryKey() {
-            return name().toLowerCase();
-        }
+@Deprecated
+public class RequestDeeplink extends RideRequestDeeplink {
```
Comment (edjiang) [inline rides-android/src/main/java/com/uber/sdk/android/rides/RequestDeeplink.java]: Confused on why you're exposing a new public init that's deprecated.

Comment (tyvsmith) [reply rides-android/src/main/java/com/uber/sdk/android/rides/RequestDeeplink.java]: The class was previously named RequestDeeplink, I renamed the impl class to RideRequestDeeplink and extended it with the old name to allow for existing customers not to break.

**Comment by bettyliew3329** [comment]
sorry can i ask for this gist request is by which one ip address asking , i think my github account got somepeople to using to change my uber driver development setup or resolved , thanks  can help to report

**Pull request merged**